Kernel/Process.cpp

```diff
@@ -4316,8 +4319,15 @@ int Process::sys$setkeymap(const Syscall::SC_setkeymap_params* user_params)
     if (!validate_read(altgr_map, 0x80))
         return -EFAULT;
 
+    for (int i = 0; i < 0x80; i++) {
+        character_map_data.map[i] = params.map[i];
+        character_map_data.shift_map[i] = params.shift_map[i];
+        character_map_data.alt_map[i] = params.alt_map[i];
+        character_map_data.altgr_map[i] = params.altgr_map[i];
```

Comment:
Since we already copy out `params.map` to `map` (and friends), it seems like we should use `map` here too.